tests/test_lang.py

```diff
@@ -9,6 +9,10 @@ def test_quote_quotes():
     assert quote('"spam"') == r'"\"spam\""'
 
 
+def test_escape_backslash():
```

Comment:
I think it might be enough to add the following cases to [the doctest](https://github.com/xflr6/graphviz/blob/499875785ccaefe334b6c1dbb45b6eaa41c2a511/graphviz/lang.py#L48) (making the docstring  a raw string):

```python
>>> print(quote('"'))
"\""

>>> print(quote('\\"'))
"\\\""
```